gpio/gpiotest/gpiotest.go

```diff
@@ -174,9 +177,9 @@ func (p *LogPinIO) Read() gpio.Level {
 
 // WaitForEdge implements gpio.PinIn.
 func (p *LogPinIO) WaitForEdge(timeout time.Duration) bool {
-	s := time.Now()
+	s := Clock.Now()
```

Comment:
I ambivalent if it's necessary, it's a different use case. In particular, I'd prefer to not have Clock be a global.